packages/flutter_tools/lib/src/emulator.dart

```diff
@@ -126,7 +126,8 @@ class EmulatorManager {
           error:
               'No suitable Android AVD system images are available. You may need to install these'
               ' using sdkmanager, for example:\n'
-              '  sdkmanager "system-images;android-27;google_apis_playstore;x86"');
+              ' sdkmanager "system-images;android-33;google_apis_playstore;x86_64"\n'
+              ' See all available images with sdkmanager --list.');
```

Comment:
did you intend to slightly un-indent the code snippet and indent the last line?

convention is to have the spaces on the right hand side, which would make the actual indentation more obvious.

Reply:
Fixed